ospfd/ospf_vty.c

```diff
@@ -7387,9 +7387,12 @@ DEFUN (show_ip_ospf_database_type_adv_router,
 		} else {
 			ospf = ospf_lookup_by_inst_name(inst, vrf_name);
 			if ((ospf == NULL) || !ospf->oi_running) {
-				vty_out(vty,
-					"%% OSPF is not enabled in vrf %s\n",
-					vrf_name);
+				if (uj)
+					vty_json(vty, json);
+				else
+					vty_out(vty,
+						"%% OSPF is not enabled in vrf %s\n",
+						vrf_name);
```

Comment:
One small suggestion,  I could see  vrf specific json pointer( json_vrf ) is not freed up in few cases in the above api (show_ip_ospf_database_type_adv_router_common). could you also please take care of this?

Reply:
Yep, will do.

Reply:
Fixed.